### PR TITLE
Adjusting ADS to support new changes for CloudServerType for SQLOD

### PR DESCRIPTION
**Pull request by BranislavGrbicMDCS**
This PR fixes:
Fixing the issues caused by Cloud Server Type changes for SQLOD.
Hiding script as create for User for SQL OD. 
Fixing the db/server dashboard since it was broken when SQL OD started representing as cloud instead standalone.

### CODE REVIEW

File: extensions/kusto/src/contextProvider.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ export enum ContextKeys {
 
 const isCloudEditions = [
 	5,
-	6
+	6,
+	11
```
Comment (Charles-Gagnon) [inline extensions/kusto/src/contextProvider.ts:24]: Let's take the opportunity to define these as constants instead so it's clear what the editions are.

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Waiting for STS update too

File: extensions/kusto/src/contextProvider.ts (hunk 1 above)
Comment (BranislavGrbicMDCS) [reply extensions/kusto/src/contextProvider.ts]: Instead of introducing new constants I've reused existing constants from azdata.DatabaseEngineEdition

**Comment by BranislavGrbicMDCS** [comment]
> Waiting for STS update too

STS update added in latest commit.

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**